index.html

```diff
@@ -1,11 +1,122 @@
+          alt="menu"></a>
+      <a id="logo" href="#home_page"><img src="images/nav/logo.png" alt="Mega Auto Auction 2022"></a>
+      <ul class="menu">
+        <li><a href="#home_page">Home</a></li>
+        <li><a href="#about_page">About</a></li>
+      </ul>
+    </nav>
+
+    <popup-menu id="popup_menu">
+      <a onclick="closeMenu()" href="#"><img src="images/popup/close.png" alt="close"></a>
+      <ul>
+        <li>
+          <a onclick="closeMenu()" href="#home_page">Home</a>
+          <hr>
+        </li>
+        <li>
+          <a onclick="closeMenu()" href="#about_page">About</a>
+          <hr>
+        </li>
+      </ul>
+    </popup-menu>
+
+    <section class="home_page" id="home_page">
+      <h3>"HELLO! AUTO WORLD"</h3>
+      <h1>MEGA AUTOMOBILE<br>AUCTION 2022</h1>
+      <P id="event_info">This is the largest online automobile auction in East Africa.<br>Top automobile dealers based
+        in Kenya,
+        Uganda, Tanzania, Ethiopia, Rwanda <br>and Burundi converge to offer the best deals.</P>
+      <p id="event_date">2022.12.1(Thur) ~ 2(Fri)</p>
+      <a id="event_location" href="#">@ auto-auction-2022.com</a>
+    </section>
+
+    <section class="about_page" id="about_page">
+      <program class="program" id="program">
+        <header id="program_header">
+          <h1>Main Program</h1>
+          <hr>
+        </header>
+        <sub-section class="program_content" id="program_content">
+          <div class="content" id="content1">
+            <p class="col_1">Thursday<br>8 - 11AM</p>
+            <h3 class="col_2">
+              <img src="images/about_page/motorcycle.png" alt="two-wheeler">
+              <br>Two Wheeler
+            </h3>
+            <p class="col_3">Motorcycles, Scooters</p>
+          </div>
+          <div class="content" id="content2">
+            <p class="col_1">Thursday<br>1 - 4PM</p>
+            <h3 class="col_2">
+              <img src="images/about_page/auto_rickshaw.png" alt="three-wheeler">
+              <br>Three Wheeler
+            </h3>
+            <p class="col_3">Tuk tuk(auto rickshow), motorized tricycles</p>
+          </div>
+          <div class="content" id="content3">
+            <p class="col_1">Friday<br>8 - 11AM</p>
+            <h3 class="col_2">
+              <img src="images/about_page/car.png" alt="four-wheeler">
+              <br>Four Wheeler
+            </h3>
+            <p class="col_3">cars, jeeps, buses, tractors</p>
+          </div>
+          <div class="content" id="content4">
+            <p class="col_1">Friday<br>1 - 4PM</p>
+            <h3 class="col_2">
+              <img src="images/about_page/truck.png" alt="six-wheeler">
+              <br>six Wheeler
+            </h3>
+            <p class="col_3">Trucks, Buses</p>
+          </div>
+        </sub-section>
+        <a href="#">Join Auto Auction 2022</a>
+      </program>
```

Comment:
- Kindly make sure that your project sticks as much as possible to the template provided. The template contains in this section 5 cards and your project contains just 4. Kindly please add one more card. :pray: :pray: 

Your design
![image](https://user-images.githubusercontent.com/98363075/203804231-6d1548da-5445-4e5c-9874-a53eb9e454df.png)

Template design
![image](https://user-images.githubusercontent.com/98363075/203804322-56b74eaf-825f-48db-8784-c82b2f2d8f02.png)

- Also please do not forget to add a hover effect. As you can see in the template when you hover over each card a white border appears around the card.

![image](https://user-images.githubusercontent.com/98363075/203804975-4afb4720-60b1-4e00-8592-e4987d40f122.png)